NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -91,6 +95,8 @@ class NeuroID
         internal var isConnected = false
         internal var locationService: LocationService? = null
 
+        internal var isAdvancedDevice = false
+
```

Comment:
This should be set in the configure/build method

Reply:
updated. thanks!